import/index.template.html

```diff
@@ -287,8 +287,12 @@
        * Function to verify enclave signature on import bundle received from the server. 
        */
       async function verifyEnclaveSignature(enclaveQuorumPublic, publicSignature, publicKey) {
-        /** Turnkey Signer enclave's public key (preprod) */
-        const TURNKEY_SIGNER_ENCLAVE_QUORUM_PUBLIC_KEY = "04f3422b8afbe425d6ece77b8d2469954715a2ff273ab7ac89f1ed70e0a9325eaa1698b4351fd1b23734e65c0b6a86b62dd49d70b37c94606aac402cbd84353212";
+        /** Turnkey Signer enclave's public keys */
+        const TURNKEY_SIGNERS_ENCLAVES = {
+          "prod": "04cf288fe433cc4e1aa0ce1632feac4ea26bf2f5a09dcfe5a42c398e06898710330f0572882f4dbdf0f5304b8fc8703acd69adca9a4bbf7f5d00d20a5e364b2569",
+          "preprod": "04f3422b8afbe425d6ece77b8d2469954715a2ff273ab7ac89f1ed70e0a9325eaa1698b4351fd1b23734e65c0b6a86b62dd49d70b37c94606aac402cbd84353212"
+        }
+        const TURNKEY_SIGNER_ENCLAVE_QUORUM_PUBLIC_KEY = TURNKEY_SIGNERS_ENCLAVES["${TURNKEY_SIGNER_ENVIRONMENT}"];
```

Comment:
Should we throw an error here if not found?

Reply:
Good call!